interceptors/retry/trace_notrace.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+//go:build retrynotrace
+
+package retry
+
+// retrynotrace can be used to avoid importing golang.org/x/net/trace, which in
+// turn enables binaries using go-grpc-middleware for dead code elimination, which can
+// yield improvements in binary size when tracing is not needed.
```

Comment:
```suggestion
// retrynotrace can be used to avoid importing golang.org/x/net/trace,
// which allows for more aggressive deadcode elimination, which can
// yield improvements in binary size when tracing is not needed.
```